src/components/Calculator.js

```diff
@@ -10,49 +10,39 @@ const content = [
   ['0', '.', '='],
 ];
 
-// eslint-disable-next-line react/prefer-stateless-function
-export class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = { obj: {} };
-    this.handleClick = this.handleClick.bind(this);
-  }
+export const Calculator = () => {
+  const [obj, setObj] = useState({});
 
-  handleClick(e) {
-    const { obj } = this.state;
-    this.setState({ obj: calculate(obj, e.target.textContent) });
-  }
-
-  render() {
-    const { obj } = this.state;
-    const { next, total } = obj;
-    return (
-      <div className="column">
+  const handleClick = (e) => {
+    setObj(calculate(obj, e.target.textContent));
+  };
+  const { next, total } = obj;
+  return (
+    <div className="column">
+      <div className="row">
+        <input type="digit" placeholder="0" value={next || total} />
+      </div>
+      {content.map((row) => (
+        // eslint-disable-next-line react/jsx-key
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practices. Instead of disabling `eslint`, it will be nice to fix the issue.